### PR TITLE
Completed html-relative-links exercise.

### PR DESCRIPTION
**Pull request by santiagolism**
Screenshot (348)

### CODE REVIEW

File: html-relative-links/html-relative-links/index.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+   <li><a href="index.html">Home Page</a></li>
+    <li><a href="../html-relative-links/about-me/about-me.html">About Me</a></li>
+    <li><a href="../html-relative-links/projects/projects.html">Projects</a></li>
+    <li><a href="../html-relative-links/contact-us/contact-us.html">Contact Us</a></li>
```
Comment (scbowler) [inline html-relative-links/html-relative-links/index.html]: Here the file is already inside the `html-relative-links` folder so you don't wan't to leave it then come back in. You can just specify the folder and the file, for example:

```HTML
<li><a href="about-me/about-me.html">About Me</a></li>
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good but a few changes needed in `index.html`. Let me know if you have any questions

**Comment by santiagolism** [comment]
Screenshot (352)

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**